New release and publish docs (#27)
operator-framework/operator-sdk-ansible-util: README.md

make test-molecule
```

### Release Docs

To perform a release, do the following steps:
1. Branch:
   - If this is a z-stream bump, checkout the minor version release branch.
   - If this is a minor version bump, create a new branch ``release-<semantic-version-number>``. For example: ``release-0.3``.

2. Change the version in Makefile and galaxy.yml to the new semantic version.

> 

Reviewer: Optional nit: Is there a reason for these extra newlines?

Author: Not really, I just thought it looked more spaced in markdown.

make test-molecule
```

### Release Docs

To perform a release, do the following steps:
1. Branch:

>    - If this is a z-stream bump, go to the minor version branch.

Reviewer: ```suggestion
   - If this is a z-stream bump, checkout the minor version release branch.
```

   - If this is a minor version bump, create a new branch ``release-<semantic-version-number>``. For example: ``release-0.3``.

2. Change the version in Makefile and galaxy.yml to the new semantic version.

3. Run ``make build`` to generate a zip of the release.

4. If it's a minor version bump, open a pull request against the master branch of the main repository.

> 5. Once the pull request in step 5 gets merged, push a tag that named by the semantic version number of this release. Ex: v0.3.1

Reviewer: ```suggestion
5. Once the pull request to master is merged, push a tag that named by the semantic version number of this release. Ex: v0.3.1
```

Reviewer: Nit: its easier to maintain if we dont reference step numbers, since they change.

> 6. Draft a release on that tag. The release notes should list the changes since the last release, as well as a zip of the current version. This zip is generated from step 3.

Reviewer: ```suggestion
6. Draft a release on that tag. The release notes should list the changes since the last release, as well as a zip of the current version. 
```

> 7. Publish the collection using the steps below.

Reviewer: can you roll up the publish command here so we only have 1 checklist?
